pulser-core/pulser/channels.py

```diff
@@ -176,6 +180,16 @@ def __post_init__(self) -> None:
             if not valid:
                 raise ValueError(msg)
 
+        if (
+            self.max_duration is not None
+            and self.max_duration < self.min_duration
```

Comment:
Ok, I wonder if this would be something to consider then: setting `min_duration` to the minimum possible value if the user doesn't set it. This way, they won't need to mind what the minimum value actual is. It will also dynamically adapt to future changes in the code in that regard (for example, if floats become allowed, or if there is the new requirement for `min_duration` to be greater than `clock_period`).

Reply:
It's a good point. I didn't touch the defaults so as to not change behaviour of existing code relying on them, but this one should be pretty harmless. Also, if there is a time to touch the defaults, it's now.

Reply:
I changed all defaults to the best possible values (with the exception of `max_duration`, which is never a limiting factor and making it `None` would unnecessarily force people to find or make up a value for it)